### PR TITLE
[TnT][WIP] Add Translation Service for TnT

### PR DESCRIPTION
**Pull request by shantanoo-desai**
- Add Translation for Tracking and Tracing Component on Frontend

- [WIP] Longer sentences are not rendered

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -99,6 +99,20 @@
 	"Overall Rating":"Overall Rating",
 	"Purchase Order":"Purchase Order",
 	"Track": "Track",
+	"Track Your Product": "Track Your Product",
+	"Enter EPC Code": "Enter EPC Code",
+	"Your SGTIN Code": "Your SGTIN Code",
+	"An Error Occured": "An Error Occured",
+	"tnt-error": "There is no information about this EPC available. Please check your code or contact your company's responsible personnel for tracking and tracing.",
```
Comment (shantanoo-desai) [inline src/assets/i18n/en.json]: @Akayeshmantha I have added a key for a long sentence error statement here.

File: src/app/tnt/tnt-form.component.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
         <div class="alert alert-danger space-before"
              role="alert">
             <p>
-                <em>An Error Occured</em> - There is no information about this EPC available. Please check your code or contact your company's responsible personnel for tracking and tracing.
+                <em><span [innerHTML]="'An Error Occured' | translate"></span></em> - <span [innerHTML]="'tnt-error' | translate"></span>
```
Comment (shantanoo-desai) [inline src/app/tnt/tnt-form.component.html]: @Akayeshmantha the single key for the long sentence error statement should be able to render the text accordingly. However this doesn't work locally

Comment (Akayeshmantha) [reply src/app/tnt/tnt-form.component.html]: Sure will check

Comment (Akayeshmantha) [reply src/app/tnt/tnt-form.component.html]: Sure will check

**Pull request merged**